third_party/terraform/tests/resource_firebase_web_app_test.go.erb

```diff
@@ -29,7 +29,8 @@ func TestAccFirebaseWebApp_firebaseWebAppFull(t *testing.T) {
 				Config: testAccFirebaseWebApp_firebaseWebAppFull(context, "2"),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrSet("data.google_firebase_web_app_config.default", "api_key"),
-					resource.TestCheckResourceAttrSet("data.google_firebase_web_app_config.default", "database_url"),
+                                        //      comment below check as database_url is not included in the api response 
+                                        //	resource.TestCheckResourceAttrSet("data.google_firebase_web_app_config.default", "database_url"),
```

Comment:
I think we should just fully remove this check instead of commenting it out. We don't really know if/when `database_url` will go back into the response

Reply:
commented lines removed